pkgs/by-name/rm/rmw/package.nix

```
@@ -30,6 +30,11 @@ stdenv.mkDerivation rec {
     ncurses
   ];
```

```suggestion
  ] ++ lib.optional stdenv.hostPlatform.isDarwin gettext;
```

allows it to build on macos.